app/components/file_attachment_list/file_attachment_document.js

```diff
@@ -382,7 +385,7 @@ const style = StyleSheet.create({
         alignItems: 'center',
         height: '100%',
         justifyContent: 'center',
-        left: 0,
+        left: parseInt(`-${circularProgressWidth}`, 10),
```

Comment:
This could be simplified to `left: -circularProgressWidth,`

Reply:
i tried simplifying it and had trouble because it does not accept string value. But this works.

Reply:
Why would `-circularProgressWidth` be a string value? Another alternative would be `-1 * circularProgressWidth`